tests/test_inference_costs.py

```diff
@@ -0,0 +1,1260 @@
+#!/usr/bin/env python3
```

Comment:
Let's lose the shebang and rename the file since it also seems to be testing retrofit costs.

also, should be rewritten with pytest.